module3/lessons/Week4/LinkedIn.md

```diff
@@ -0,0 +1,158 @@
+---
+
+### Learning Goals
+By the end of this lesson, you should have accomplished these learning goals:
+* Understand how to tell your story through LinkedIn
+* Know how to build a working draft of your LinkedIn profile with these components:
+  * Headshot / Cover Photo
+  * Headline updated to include your software identity
+  * Summary section to tell your professional story
+  * Experience
+
+### Section 1: Introduction
+
+❓What do you already know about LinkedinIn? Have you used it before?
+
+LinkedIn is a social media tool you can use to:
+* Tell your career story on digital platforms
+* Communicate with recruiters, peers, and thought-leaders in your industry
+* Connect with people to build your network
+* Search for and apply to jobs online
+* Research companies, employees, and available opportunities
+
+<section class="call-to-action" markdown="1">
+Create a [Linkedin](https://www.linkedin.com/feed/) Profile if you don't yet have one. Then find a LinkedIn profile that you like. What's one thing you like about it? 
+</section>
+
+### Section 2: Profile and Photos Overview
+Per Linkedin, a complete profile has a current industry listed, at least 2 up-to-date positions with descriptions, a minimum of 3 skills, a profile picture, and at least 50 connections.
+
+* Profile Photo: Your profile photo should be easy to see your face and not cropped, blurry, or pixelated.
+
+* Cover Photo: The cover photo should showcase some of your personality or connect to your industry. Above all else, replace the standard Linkedin image. [Unsplash](https://unsplash.com/) has tons of free high quality images.
+
+![Linkedin Example 1](/assets/images/module3/Week4/LinkedinExample1.png)
+![Linkedin Example 2](/assets/images/module3/Week4/LinkedinExample2.png)
+
+More examples can be found [here](https://zapier.com/blog/linkedin-banner-ideas/).
+
+### Section 3: Headlines and Keywords
+When searching on Linkedin, your profile picture and headline are some of the first things people can see about you. They can inspire people to click on your profile and learn more about you. 
+
+#### There are 3 typical types of Headlines on Linkedin: 
+* Direct = `Title of position at Company`
+* Power Statement = `title | benefit statement`
+* Keyword Focused = `Action | General title | General title `
+
+**Keyword Focused** is the best option for job seekers as concentrated keywords can help attract specific individuals to your profile. If you’re looking for a job in software development but your headline and profile only showcase your previous industry experience in education or pre, how will recruiters know to connect with you about work in software development?
+
+#### Examples
+* `Software Engineer - UI/Java/JavaScript Developer - Front End`
+
+* `Software Development Student | Seeking Full-Time Software Engineering Job | Java | C++ | 2020 Graduation`
+
+* `Aspiring Java Software Developer | Seeking Entry-Level Programming Position | Experience with Javascript and Python`

```

Comment:
What do you think about making these examples (or at least one) based in `.NET` or `C#` ?

Reply:
My original thought was that if I had one with .NET or C# they would just copy that exactly, but I see you point. I'll add some C# in there :)